examples/tag/subscription/Subscription.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Threading;
+using NationalInstruments.SystemLink.Clients.Tag;
+using NationalInstruments.SystemLink.Clients.Tag.Values;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Tag
+{
+    /// <summary>
+    /// Example for the SystemLink Tag client API that uses a subscription to
+    /// receive tag value updates.
+    /// </summary>
+    class Subscription
+    {
+        /// <summary>
+        /// The number of tags created for the example.
+        /// </summary>
+        private const int NumTags = 2;
+
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            /*
+             * Create the TagManager for communicating with the server and the
+             * tag writer for sending tag values to the server.
+             */
+            using (var manager = new TagManager(configuration))
+            using (var writer = manager.CreateWriter(NumTags * 2))
+            {
+                /*
+                 * Create two tags, one a double and the other an int.
```

Comment:
Could we make this into two sentences? "Create two tags. Make one a double and the other an integer."?

Reply:
Sounds good. Note that I specifically used int instead of integer to match the C# variable type.

Reply:
Ah! Got it.